04-materials/02-anatomy-of-extensions.md

```diff
@@ -44,14 +64,34 @@ graph TB
 ```
 
 
-## Exercise A: Create an extension from the [official template](https://github.com/jupyterlab/extension-template)
+## Now we're ready to build an extension!
+
+First, let's look at
+[the extension we're going to build today](https://github.com/jupytercon/jupytercon2025-developingextensions-demo).
+The README in this repository describes the functionality we will build out.
+
+First we'll look at the final extension together. It:
+
+* Adds a new button to the launcher
+* Adds a new command to the command palette
+* When either of those is triggered, it opens a new tab/window with a viewer showing a
+  random image and caption from a small curated collection of public domain images.
+* The viewer also has a "refresh" button to trigger fetching a new image and caption.
+
+Now, let's build it together from scratch.
+
+
+## Exercise A (10 minutes): Extension creation and development loop
+
+### First, create a new extension from the [official template](https://github.com/jupyterlab/extension-template)
 
 :::{important} Prerequisites
 * `copier`. Install with any of the following options:
```

Comment:
Let's have a recommended way to install, then other ways if they are comfortable. Otherwise people are going to be paralyzed by the choices here.

Reply:
Yeah, let's go with all-conda approach. I'll implement that in a separate PR.